src/stratigraphy/annotations/draw.py

```diff
@@ -50,7 +48,7 @@ def draw_predictions(
         predictions (dict): Content of the predictions.json file.
         directory (Path): Path to the directory containing the pdf files.
         out_directory (Path): Path to the output directory where the images are saved.
-        document_level_metadata_metrics (pd.DataFrame): Document level metadata metrics.
+        document_level_metadata_metrics (None | pd.DataFrame): Document level metadata metrics.
```

Comment:
Because the method evaluate() in score.py, where the value for `document_level_metadata_metrics` comes from, might return `None` in case there is no ground truth available.

We could consider changing that into returning an empty dataframe instead of None. However, I think that such changes are a bit out of scope for this PR.